toc.json

```diff
@@ -94,6 +94,9 @@
       "sanitizers": "Sanitizers"
     }
   },
+  "configuration_file": {
+    "name": "Configuration file"
+  },
```

Comment:
I don't think this should be a top level key. Maybe part of `Getting started`, like `Command line interface`?